rtengine/dcrop.cc

```diff
@@ -1171,6 +1171,10 @@ void Crop::update(int todo)
         bool cclutili = parent->cclutili;
 
         LUTu dummy;
+        bool hasColorToningLabGrid = params.colorToning.enabled && params.colorToning.method == "LabGrid";
```

Comment:
Why create a new variable? This could be directly part of the if () statement below.